src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,82 @@
-import React from 'react';
+import {
+  FC,
+  FormEvent,
+  useCallback,
+  useState,
+} from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+interface FindMovieProps {
+  addMovie: (movie: Movie) => void;
+}
+
+const FILM_URL = 'https://www.imdb.com/title';
+// eslint-disable-next-line max-len
+const DEFAULT_PICTURE = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+export const FindMovie: FC<FindMovieProps> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [hasError, setError] = useState(false);
+  const [isFindedFilm, setIsFindedFilm] = useState(false);
+  const [isSubmitedForm, setSubmitedForm] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const changeQuery = (queryFromUser: string) => {
+    setQuery(queryFromUser);
+    setError(false);
+  };
+
+  const loadMoviesFromServer = useCallback(async (queryFromUser) => {
+    const movieFromServer = await getMovie(queryFromUser);
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`